File: TP1 python/Cesar/tp1/punto4/Categoria.py
class Categoria:
    def __init__(self, titulo):
        self.__titulo = titulo
        self.__Articulo = []
        
    def agregar_articulo(self,Articulo):
        self.__Articulo.append(Articulo)
    
    def imprimir(self):
        print(self.__titulo)
        for pepe in self.__Articulo:
            pepe.imprimir()
        
    

File: Ejemplos de ejercicios/Ejercicio 3 TP1/Java/Categoria.java

import java.util.ArrayList; // importacion de la clase ArrayList

public class Categoria { // clase Categoria
    
    private String titulo; // titulo de la categoria
    private ArrayList<Serie> series; // lista de series


    public Categoria(String titulo){
        this.titulo = titulo;
        this.series = new ArrayList<>();
    }

    public void agregarSerie(Serie ser){
        this.series.add(ser);
    }

    public void imprimir(){
        System.out.println(this.titulo);
        for(Serie ser:this.series){
            ser.imprimir();
        }
    }

}
